pyseir/inference/infer_rt.py

```diff
@@ -502,17 +507,39 @@ def run_state(state, states_only=False):
     state_obj = us.states.lookup(state)
     df = RtInferenceEngine.run_for_fips(state_obj.fips)
     output_path = get_run_artifact_path(state_obj.fips, RunArtifact.RT_INFERENCE_RESULT)
-    df.to_json(output_path)
+    with open(output_path, 'w') as buf:
```

Comment:
Why this? Pandas already does this for you.

Reply:
Slight i/o improvement.  The less pandas has to infer, the better